packages/dds/tree/src/core/schema-stored/schema.ts

```diff
@@ -185,18 +214,41 @@ export class ObjectNodeStoredSchema extends TreeNodeStoredSchema {
 		super();
 	}
 
-	public override encode(): TreeNodeSchemaDataFormat {
+	public override encodeV1(): TreeNodeSchemaDataFormatV1 {
 		const fieldsObject: Record<string, FieldSchemaFormat> = Object.create(null);
 		// Sort fields to ensure output is identical for for equivalent schema (since field order is not considered significant).
 		// This makes comparing schema easier, and ensures chunk reuse for schema summaries isn't needlessly broken.
 		for (const key of [...this.objectNodeFields.keys()].sort()) {
+			const value = encodeFieldSchemaV1(
+				this.objectNodeFields.get(key) ?? fail(0xae6 /* missing field */),
```

Comment:
Curious why the update to the hex code or if it was a mistake. Since this is the same codepath as the only one before the PR, I think keeping the same hex code makes sense here.
```suggestion
				this.objectNodeFields.get(key) ?? fail(0xae7 /* missing field */),
```